packages/docs/blog/2025-04-10-Release-v0.12.0.mdx

```diff
@@ -0,0 +1,53 @@
+---
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+slug: v0.12.0
+title: Release 0.12.0
+authors: [mellyeliu, necolas]
+tags:
+  - release
+---
+
+# Release 0.12.0
+
+StyleX v0.12.0 introduces performance optimizations, several bug fixes, a new lint rule, and some cleanup of deprecated packages.
+
+## Performance
+
+### Hashing Style Object Keys
+
+We added a new config option to minify keys in compiled style objects to reduce overall code size. To enable, turn on `enableMinifyKeys` within your project's `.babelrc` file.
```

Comment:
This is actually on by default. I don't think we should advertise these feature flags either, as they're mainly there to help us roll out changes internally, not stable parts of the public API

Reply:
https://github.com/facebook/stylex/pull/992